src/antithesis_sdk/assertions.py

```diff
@@ -360,6 +361,11 @@ def assert_raw(
         assert_id,
     )
 
+assertion_catalog = os.getenv(ASSERTION_CATALOG_ENV_VAR)
+if assertion_catalog is not None:
+    execfile(assertion_catalog)
```

Comment:
So if we have an assertion catalogue, we will register it when we load up the assertions module. 

Sounds good to me so far